std/lua/_std/String.hx

```diff
@@ -31,38 +31,31 @@ class String {
 	public var length(default,null) : Int;
 
 
-	public function new(string:String) untyped {}
+	public inline function new(string:String) untyped {}
```

Comment:
unfortunately, this won't work as-is with  ``--no-inline``, even with ``@:extern``.